priv/yokozuna.schema

```diff
@@ -338,6 +338,19 @@
   hidden,
   {validators, ["positive_integer"]}]}.
 
+%% @doc Enable or disable this node in distributed query plans.  If enabled,
+%% this node will participate in distributed Solr queries.  If disabled, the
+%% node will be excluded from yokozuna cover plans, and will therefore never
+%% be consulted in a distributed query.  Note that this node may still be used
+%% to execute a query.  Use this flag if you have a long running administrative
+%% operation (e.g., reindexing) which requires that the node be removed from
+%% query plans, and which would otherwise result in inconsistent search results.
+{mapping, "search.dist_query.enable", "yokozuna.enable_dist_query", [
```

Comment:
Does that preclude 'on' and 'off' or 'true' and 'false'?  I thinks CSEs prefer that flexibility.  But I agree we should remove the .enable